### PR TITLE
Add exports for metingen and referentiepunten

### PR DESCRIPTION
**Pull request by jasper-jager**
Export for meetbouten catalog were added and made generic using a export config containing format and the API path. Documentation was updated as well as the docker-compose file to match the current status.

### CODE REVIEW

File: src/export/meetbouten/__init__.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+def _export_entity(entity, format):
+    """Exports a single entity
+
+    The export file specifications can be found in:
+        https://www.amsterdam.nl/stelselpedia/meetbouten-index/producten/prodspecs-mebo/
+
+    Headers:       None
+    Separator:     |
+    String marker: $$
+
```
Comment (Vegter) [inline src/export/meetbouten/__init__.py]: This info is no longer appropriate for this method

File: src/export/meetbouten/__init__.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    pattern = re.compile('(\w+):(\w+)\|?')
+    export = format
+    for (attr_name, attr_type) in re.findall(pattern, format):
+        attr_value = type_convert(attr_type, entity.get(attr_name, None))
+        export = export.replace(f'{attr_name}:{attr_type}', attr_value)
+    return export
+
+
+def export_meetbouten(collection, host, file):
+    """Export a collection from catalog meetbouten to a file
```
Comment (Vegter) [inline src/export/meetbouten/__init__.py]: Why is meetbouten still in the name?

File: src/export/meetbouten/config.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        referentiepuntid: 25280009:
+        http://localhost:5000/gob/meetbouten/referentiepunten/25280009/
+        =>
+        $$25280009$$|121155|489090|2,2005|$$19790101$$|$$Zoutkeetsgracht 2 hoek Bokkinghangen$$
+        |POINT (121155.0 489090.0)
+
+    """
+    format = 'referentiepuntid:num|geometrie:xco|geometrie:yco|' \
+             'hoogte_tov_nap:num|datum:dat|locatie:str|geometrie:geo'
+    path = '/gob/meetbouten/referentiepunten/'
```
Comment (Vegter) [inline src/export/meetbouten/config.py]: Nice configs!

File: src/export/meetbouten/types.py [hunk 4]
```diff
@@ -97,6 +98,58 @@ def _to_geometry(value):
...
+                487201.6
+            ]
+        }
+        Output: 119411,7
+
+    :param value:
+    :return:
+    """
+    assert(type(value) is dict or value is None)
+    return _get_coord(value, 0)
```
Comment (Vegter) [inline src/export/meetbouten/types.py]: It is a liitle confusing that None results in ''.
Probably move the return '' outside _get_coord or add a parameter when_none='' or something like that

File: src/export/meetbouten/__init__.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    :param host: The API host
+    :param file: The name of the file to write the ouput
+    :return: None
+    """
+    config = CONFIG_MAPPING[collection]
+    api = API(host=host, path=config.path)
+
+    with open(file, 'w') as fp:
+        for entity in api:
+            fp.write(_export_entity(entity, config.format) + '\n')
```
Comment (Vegter) [inline src/export/meetbouten/__init__.py]: You can consider taking this code out of init to export.py or something like that.
Is mannot it not be called more generic, like export or so?

Comment (jasper-jager) [reply src/export/meetbouten/__init__.py]: Discussed to take into account on next catalog export

File: src/export/meetbouten/__init__.py (hunk 2 above)
Comment (jasper-jager) [reply src/export/meetbouten/__init__.py]: Discussed to take into account on next catalog export

File: src/export/meetbouten/__init__.py (hunk 1 above)
Comment (jasper-jager) [reply src/export/meetbouten/__init__.py]: Discussed to take into account on next catalog export

**Pull request merged**